game/src/edit/roads.rs

```diff
@@ -398,19 +399,43 @@ fn make_main_panel(
     let available_lane_types_row = Widget::row(available_lane_types_row);
 
     let mut current_lanes_ltr = Vec::new();
-    for (id, _, lt) in road.lanes_ltr() {
-        // TODO Add direction arrow sometimes
+    for (id, dir, lt) in road.lanes_ltr() {
+        let mut stack = GeomBatchStack::vertical(vec![
+            Image::from_path(lane_type_to_icon(lt).unwrap())
+                .build_batch(ctx)
+                .unwrap()
+                .0,
+        ]);
+        if can_reverse(lt) {
```

Comment:
Note this returns true for everything right now, which looks a little weird. We need to automatically manage direction for parking lanes and sidewalks and stuff, based on which side of the road these lanes are. Until that's fixed, explicitly showing the arrow lets people fix bugs like this:
![Screenshot from 2021-04-28 16-31-10](https://user-images.githubusercontent.com/1664407/116485032-26a9cb80-a83f-11eb-8cee-baa4a9886cf6.png)